src/librustdoc/html/highlight/fixtures/decorations.html

```diff
@@ -1,4 +1,4 @@
-<span class="example"><span class="kw">let </span>x = <span class="number">1</span>;
+<span class="kw">let </span>x<span class="example"> = <span class="number">1</span>;
```

Comment:
This change highlights (hehe) the fact that now, we open the tag only when we push content to it. Same look but slightly different DOM.